wp-content/themes/illdy-child/layout/js/translations_page.js

```diff
@@ -165,25 +193,26 @@ var TranslationsPage = (function(window, $) {
       filter.setAttribute('type', 'text');
       filter.setAttribute('placeholder', 'Search languages or codes...');
 
+      var resource_filter = create('select', 'lang-list-resource-filter', 'lang-list-resource-filter');
+      this.createResourceFilterEls().forEach(function(item) {
+          resource_filter.add(item);
+      });
+      var filterLangs = this.filterLangs;
+      var translationsData = this.list;
+      resource_filter.onclick = function() {

```

Comment:
This should probably onchange for the same reason as above.

Reply:
Yeah, I wanted to use `onChange()` here but it doesn't work in IE.  :(